canonical-websites/snapcraft.io
# Rename public namespace to store

# Summary

Previously called `public` I renamed the namespace to `store`. Public made sens in our previous architecture, now that everything has been well splitted in different modules the module public was only providing store views.

# QA

- `./run`
- Check that those routes still work:
```
snapcraft.homepage                      GET        /
store.discover                          GET        /discover
store.search_snap                       GET        /search
store.snap_details                      GET        /toto
store.store_view                        GET        /store
```
